lib/cli.js

```diff
@@ -164,6 +164,10 @@ class CLI {
       this.log(prefix + chalk.bold(`${error}  ${obj}`));
     }
   }
+
+  setErrorCode(statusCode = 1) {
```

Comment:
it might be a tad clearer if this was `setExitCode(statusCode)` and then we passed `1` explicitly - i'd probably not be 100% sure what this did without checking the definition i don't think 🤔